src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2121,9 +2130,16 @@ Json::Value NetworkOPsImp::getServerInfo (bool human, bool admin)
     }
     else
     {
-        info[jss::load_factor] =
-            static_cast<double> (app_.getFeeTrack ().getLoadFactor ()) /
-                app_.getFeeTrack ().getLoadBase ();
+        auto const loadBase = app_.getFeeTrack().getLoadBase();
+        assert(!escalationMetrics ||
+            loadBase == escalationMetrics->referenceFeeLevel);
+
+        info[jss::load_factor] = static_cast<double> (loadFactor) / loadBase;
```

Comment:
I'm taking it on faith that the numbers you're reporting for `load_factor`, `load_factor_server` and `load_factor_fee_escalation` are the ones you want.